deepa/Schedule/ScheduleView.swift

```diff
@@ -18,15 +19,55 @@ struct ScheduleView: View {
         _viewModel = StateObject(wrappedValue: viewModel)
         
     }
+
+    static let extractDate: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "dd/MM/yyyy"
```

Comment:
this can be named as just `dateFormatter` 
and use `dateFormatter.setLocalizedDateFormatFromTemplate("dd/MM/yyyy")` so that it will appear nicer according to the user's locale
see https://www.wwdcnotes.com/notes/wwdc20/10160/